assets/icons/google_icon.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react"
+import Svg, { Path } from "react-native-svg"
+
+function SvgComponent(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
Rename function to GoogleIcon

And change the icon props to use the proper icon props (IconProps)

*EXAMPLE FROM HAIR_CARE_ICON INSIDE ICON FOLDER

```javascript
import { IconProps } from './types';

export default function HairCareIcon(props: IconProps) {
  return (
    <Svg width="100%" height="100%" viewBox="0 0 58.53 58.53" {...props}>
      <Path
        d="M52.551 23.053c-.939-9.311-7.074-20.351-18.323-22.59A24.357 24.357 0 0029.484 0C21.411 0 13.691 3.871 9.336 10.102c-.11.157-.217.315-.323.476-4.832 7.321-4.076 17.498 1.721 23.168.432.422.891.824 1.349 1.225 1.089.952 2.118 1.852 2.788 3.027 1.099 1.926 1.111 4.601.032 7.157-1.001 2.37-2.714 4.381-4.371 6.327a.5.5 0 00.413.823c3.791-.238 7.421-2.294 9.959-5.641 2.537-3.347 3.537-7.397 2.743-11.112a.497.497 0 00-.385-.384.498.498 0 00-.508.193c-.534.728-1.33.881-1.903.881-.914 0-1.919-.398-2.56-1.014-3.168-3.041-3.391-7.813-3.242-10.387.18-3.116 1.245-5.5 2.373-8.024l.104-.232c.525 2.546 2.638 4.906 4.753 6.427.544.392 1.108.773 1.676 1.157 2.388 1.612 4.857 3.28 5.848 5.854 1.474 3.828-.786 8.026-3.447 11.993-.104.154-.098.351-.011.514s.272.265.457.265c1.226 0 2.449-.458 3.396-1.239-.075 4.447-4.737 7.962-8.365 10.278l-8.626 5.507a.499.499 0 00.199.917c1.288.181 2.61.273 3.929.273 9.545 0 19.716-4.817 27.907-13.217 5.341-5.481 8.142-14.011 7.309-22.261z"
        fill={props.fillColor}
      />
    </Svg>
  );
}


```